### ISSUE TITLE
issue.py does not seem to support 'start_id' flag

### ISSUE THREAD

**Issue by MastaP** [ttencate/sf2github#30]
I ran the script for the first time, it successfully migrated 25 issues and then GH started throwing 403s
```
403: You have triggered an abuse detection mechanism and have been temporarily blocked from content creation. Please retry your request again later.
```
So I killed the script execution with ctrl+c.
Before continuing I opened the code just to check if 'start_id' is expecting a value from 'ticket_num' field and to my surprise I did not find anything that binds these two.
I did not study the code enough yet and I'm kinda afraid of running it for the second time and potentially getting a bunch of duplicates...
Plz correct me if I'm wrong

**Comment by MastaP** [comment]
OK, I've made a copy of my bugs.json, removed all but one ticket that has already been migrated. Executing the script produced a duplicate issue in GH (I was expecting/hoping the issue would be updated).

**Comment by ttencate** [comment]
The script has no way of resolving/avoiding duplicates (PR welcome!).

If GitHub have added some new abuse mechanism, I think batching your
uploads in groups of 25 would be your best bet.

**Comment by MastaP** [comment]
I added sleep(3) into each loop and the script was able to migrate the rest of issues (I had to manually remove already imported ones) without triggering the abuse mechanism.

**Comment by codeguru42** [comment]
[Rate Limiting](https://developer.github.com/v3/#rate-limiting) in the GitHub API v3 documentation. This may be helpful in trying to fix this problem. My suggestion is to add a delay when a 403 error message is received. This seems more desirable than adding a delay before *every* call to the GitHub API. Are there any other suggestions for fixes?

**Comment by codeguru42** [comment]
As for resolving duplicate messages, we should start a separate issue and/or pull request.

**Comment by bebehei** [comment]
I would assume, this is already fixed with #33. IMO I would close it. @MastaP test it again with the current master. I would guess, you won't experience these errors anymore.
